google-cloud-bigtable/src/test/resources/logging-verbose.properties

```diff
@@ -0,0 +1,17 @@
+handlers= java.util.logging.ConsoleHandler
+.level= FINEST
+
+java.util.logging.ConsoleHandler.level = FINEST
+java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter
+io.grpc.level=FINEST
```

Comment:
since you are configuring the root logger to FINEST you dont need to set the grpc level